flink-java/src/main/java/org/apache/flink/api/java/ExecutionEnvironment.java

```diff
@@ -580,6 +580,14 @@ public CsvReader readCsvFile(String filePath) {
 	}
 
 	/**
+	 * Creates a {@link DataSet} from {@link org.apache.hadoop.mapred.SequenceFileInputFormat}
+	 * A {@link org.apache.hadoop.mapred.JobConf} with the given inputPath is created.
+ 	 */
+	public <K,V> DataSource<Tuple2<K, V>> readAsSequenceFile(Class<K> key, Class<V> value, String inputPath) throws IOException {
```

Comment:
Can we rename the method to `readSequenceFile()` to have it consistent with the other methods (`readCsvFile`, `readHadoopFile`, ...)